protocol/common/message.go

```diff
@@ -632,6 +632,17 @@ func (m *Message) PrepareContent(identity string) error {
 	return m.parseAudio()
 }
 
+func (m *Message) IsSystemMessage() bool {
+	return m.ContentType == protobuf.ChatMessage_SYSTEM_MESSAGE_CONTENT_PRIVATE_GROUP ||
+		m.ContentType == protobuf.ChatMessage_SYSTEM_MESSAGE_GAP ||
+		m.ContentType == protobuf.ChatMessage_CONTACT_REQUEST ||
```

Comment:
Not sure Contact request is needed here, it's more like a plain text message

Reply:
Is `ChatMessage_CONTACT_REQUEST` the message that the user wrote when sending the request or another message generated by the system?

Reply:
Right. I don't think that qualifies as system message. I'll remove it.